src/components/AddBook/addbook.js

```diff
@@ -23,24 +23,28 @@ const AddBook = () => {
 
   return (
     <form onSubmit={submitHandler}>
-      <p>ADD NEW BOOK</p>
-      <input
-        type="text"
-        name="title"
-        value={title}
-        onChange={(e) => setTitle(e.target.value)}
-        placeholder="Book title"
-      />
-      <input
-        type="text"
-        name="author"
-        value={author}
-        onChange={(e) => setAuthor(e.target.value)}
-        placeholder="Book author"
-      />
-      <button type="submit" className="btn-submit">
-        Add Book
-      </button>
+      <p className="form-title">ADD NEW BOOK</p>
+      <div className="form">
+        <input
```

Comment:
## Zeplin's Form

![image](https://user-images.githubusercontent.com/93039359/230583963-5b7fd439-4bce-40cc-85a9-97eeb66795a1.png)

### Your Form
![image](https://user-images.githubusercontent.com/93039359/230584225-5494f541-152b-4ca1-84fb-555464f3f863.png)

- [ ] It would be great if the **ADD NEW BOOK** should be aligned with the book items as you can see in the Zeplin's design above ⬆️ which is requested by the previous reviewer 😊